test/parallel/test-repl-tab.js

```diff
@@ -0,0 +1,50 @@
+var assert = require('assert');
+var util = require('util');
+var repl = require('repl');
+var zlib = require('zlib');
+var defaultScopes = [
```

Comment:
This is going to fail if there will be new globals introduced :) Let's skip this assert.